cmp/dataview/DataViewModel.ts

```diff
@@ -107,13 +108,13 @@ export interface DataViewConfig {
      * Callback when a row is clicked - will receive an event with a data node containing
      * the row's data. (Note that this may be null - e.g. for clicks on full-width group rows.)
      */
-    onRowClicked?: (e: any) => void;
+    onRowClicked?: (e: RowClickedEvent) => void;
 
     /**
-     * Callback when a row is double clicked - will receive an event with a data node containing
+     * Callback when a row is double-clicked - will receive an event with a data node containing
      * the row's data. (Note that this may be null - e.g. for clicks on full-width group rows.)
      */
```

Comment:
we are exposing the underlying ag-Grid implementation of DataView, but I don't think we have a choice